src/index.html

```diff
@@ -11,12 +11,52 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="./images/logo.png"
+          alt="logo moyo"
+        />
+      </a>
+      <nav class="nav--bar">
+        <ul class="nav--links">
+          <li class="nav--link is-active"><a href="#">Apple</a></li>
+          <li class="nav--link"><a href="#">Sansung</a></li>
```

Comment:
There is a typo in the navigation link text 'Sansung'. It should be corrected to 'Samsung'.